api/config/config_staging.json

```diff
@@ -81,7 +81,7 @@
     "credentialsKeyV2": "moodle-key-v2.txt"
   },
   "access": {
-    "enableComplianceTraining": false,
+    "enableComplianceTraining": true,
```

Comment:
This may have impacts on Rainforest. Requirements here would be to (1) confirm it's possible to complete compliance training in our staging environment (we may not have set up the Moodle keys necessary for API integration, so there may be errors), and (2) confirm with Rohini and Alex that no tests should be impacted.

The requirements for environment consistency for access modules are not as strict as for feature flags, since access modules are designed to differ across environments and to be potentially very long-lived as optional modules in the system. So, depending on 1 & 2 above, it may be prudent to just keep it turned off in staging.

Reply:
OK - I'll revert